docs/marketplace/add_to_marketplace/marketplace_item_visibility_conversion.md

```diff
@@ -0,0 +1,32 @@
+---
+id: marketplace_items_visibility_conversion
+title: How to change a Marketplace item's visibility from private to public and vice versa
```

Comment:
```suggestion
title: How to change a Marketplace item visibility from private to public and vice versa
```

better without possessive form since visibility is an attribute and not that is possessed by the marketplace item 🤔